docs/modules_management.rst

```diff
@@ -0,0 +1,194 @@
+
+In the variable ``sys.path`` there is a directory ``site-packages`` which contains the installed **external packages**,
+which means you can install packages with ``pip`` or ``anaconda`` and you can use them in Airflow. In the next section,
+you will learn how to create your own simple installable package and how to specify additional directories to be added
+to ``sys.path`` using the environment variable :envvar:`PYTHONPATH`.
+
+
+Creating a package in Python
+----------------------------
+
+1. Before starting, install the following packages:
+
+``setuptools``: setuptools is a package development process library designed for creating and distributing Python packages.
+
+``wheel``: The wheel package provides a bdist_wheel command for setuptools. It creates .whl file which is directly
+installable through the ``pip install`` command. We can then upload the same file to pypi.org.
+
+.. code-block:: bash
+    pip install --upgrade pip setuptools wheel
+
+2. Create the package directory - in our case, we will call it ``airflow_operators``.
+
+.. code-block:: bash
+    mkdir airflow_operators
+
+3. Create the file ``__init__.py`` inside the package and add following code:
+
+.. code-block:: python
+
+    print("Hello from airflow_operators")
+
+When we import this package, it should print the above message.
+
+4. Create ``setup.py``:
+
+.. code-block:: python
+
+    import setuptools
+
+    setuptools.setup(
+        name='airflow_operators',
+    )
+
+5. Build the wheel:
+
+.. code-block:: bash
+
+    python setup.py bdist_wheel
+
+This will create a few directories in the project and the overall structure will look like following:
+
+.. code-block:: bash
+
+    .
+    ├── airflow_operators
+    │   ├── __init__.py
+    ├── airflow_operators.egg-info
+    │   ├── PKG-INFO
+    │   ├── SOURCES.txt
+    │   ├── dependency_links.txt
+    │   └── top_level.txt
+    ├── build
+    │   └── bdist.macosx-10.15-x86_64
+    ├── dist
+    │   └── airflow_operators-0.0.0-py3-none-any.whl
+    └── setup.py
+
+
+6. Install the .whl file using pip:
+
+.. code-block:: bash
+
+    pip install dist/airflow_operators-0.0.0-py3-none-any.whl
+
+7. The package is now ready to use!
+
+.. code-block:: pycon
+
+  >>> import airflow_operators
+  Hello from airflow_operators
+  >>>
+
+The package can be removed using pip command:
+
+.. code-block:: bash
+
+    pip uninstall airflow_operators
+
+For more details, see: https://packaging.python.org/tutorials/packaging-projects/
+
+
+Adding directories to the path
+------------------------------
+
+You can specify additional directories to be added to ``sys.path`` using the environment variable :envvar:`PYTHONPATH`.
+Start the python shell by providing the path to root of your project using the following command:
+
+.. code-block:: bash
+
+    PYTHONPATH=/home/arch/projects/airflow_operators python
+
+The ``sys.path`` variable will look like below:
```

Comment:
``airflow info`` please.

Reply:
I've added airflow info details example below this as well.